internal/service/dax/sweep.go

```diff
@@ -25,19 +26,20 @@ func sweepClusters(region string) error {
 	ctx := sweep.Context(region)
 	client, err := sweep.SharedRegionalSweepClient(ctx, region)
 	if err != nil {
-		return fmt.Errorf("Error getting client: %s", err)
+		return fmt.Errorf("Error getting client: %s", err.Error())
 	}
-	conn := client.DAXConn(ctx)
+	conn := client.DAXClient(ctx)
 
-	resp, err := conn.DescribeClustersWithContext(ctx, &dax.DescribeClustersInput{})
+	resp, err := conn.DescribeClusters(ctx, &dax.DescribeClustersInput{})
 	if err != nil {
 		// GovCloud (with no DAX support) has an endpoint that responds with:
 		// InvalidParameterValueException: Access Denied to API Version: DAX_V3
-		if awsv1.SkipSweepError(err) || tfawserr.ErrMessageContains(err, "InvalidParameterValueException", "Access Denied to API Version: DAX_V3") {
+
+		if awsv2.SkipSweepError(err) || errs.IsA[*awstypes.InvalidParameterValueException](err) {
```

Comment:
Im not sure if this error check is the same due to the string check.

Reply:
@danquack there is an equivalent function: `errs.IsAErrorMessageContains()`